test/driver.js

```diff
@@ -66,6 +66,51 @@ function writeSVG(svgElement, ctx, resolve, reject) {
   };
 }
 
+function inlineImages(images) {
+  const imagePromises = [];
+  for (let i = 0, ii = images.length; i < ii; i++) {
+    const imagePromise = new Promise(function (resolve, reject) {
+      const xhr = new XMLHttpRequest();
+      xhr.responseType = "blob";
+      xhr.onload = function () {
+        const reader = new FileReader();
+        reader.onloadend = function () {
+          resolve(reader.result);
+        };
+        reader.readAsDataURL(xhr.response);
+      };
+      xhr.onerror = function (e) {
+        reject(new Error("Error fetching inline annotation image " + e));
```

Comment:
Nit: Let's remove the "annotation " part, since the function is now used more generally.